src/components/navbar/SignedInLinks.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+
+const signedInLinks = () => (
+<div>
+<ul className="nav navbar-nav mr-auto">
+    <li className="nav-item">
+        <NavLink className="nav-link" activeClassName="active" to="#">
+        New Event
+        </NavLink>
+    </li>
+    <li className="nav-item">
+        <NavLink className="nav-link" activeClassName="active" to="#">
+        Log out
+        </NavLink>
+    </li>
+    <li className="nav-item">
+        <NavLink className="nav-link" activeClassName="active" to="#">
+        Oceane
```

Comment:
why did you hardcode this

Reply:
I intend to do the `signedInLinks` and `signedOutLinks` navbar implementation in a PR of its own. I had too many changed files for this one. needed to keep track.